### DISCUSSION TITLE
記事を複数のOrganizationに紐づけしたい

### DISCUSSION THREAD

**Discussion by two2mi** [increments/qiita-discussions#264] [要望・アイデア・不具合]
●要望の背景
会社でOrganizationを作成して技術情報を発信することを検討しています。
ただ、社外のエンジニアコミュニティにも所属しており、双方に関連する記事を書く場合にどちらに紐づけるか悩むことになりそうです。
そのため、可能であれば記事投稿時に複数のOrganizationに紐づけしたいです。
※現在、複数のOrganizationに所属できますが、投稿に紐づけられるOrganizationは1つになっているかと思います。

実装方式はご検討いただければと思いますが、案を考えてみました。

●変更案
記事の投稿時の「Organizationの紐付け」をプルダウンからチェックボックスに変更する

●チェックボックス化によるメリット
・記事投稿時に複数のOrganizationに紐付けられることが直感的に分かりやすい。

●チェックボックス化により考えられるデメリット
・所属しているOrganizationが多い人にとって、チェックボックスの場合は投稿時のポップアップが縦に長くなってしまう。
・多数のOrganizationを紐づけられた投稿記事の場合、（利用者の情報とOrganizationの情報が長くなることで）読み手にとって記事末尾からコメントまでの幅が長くなってしまう。※紐付けられるOrganization数を2か3を上限とするなどで対処

**Comment by tanaka-asami415** [comment]
@two2mi 
ご意見ありがとうございます🙏
現在Organizationの改善に取り組んでおりまして、必ず実施するとはお伝えできないのですが、
貴重な意見として参考にさせていただきます🙇
ぜひ今後もご意見をいただけますと幸いです！

**Reply by two2mi** [reply]
ご回答ありがとうございます！
Organizationは他サービスではあまり見られない機能なので、より良いものにしていっていただけると嬉しいです。
※企業のテックブログとしては誰が書いているかという「個人の顔」が見えたほうが「この人と働きたい」という求人にも繋がりますので、ぜひ取り入れていただけると嬉しいです。

Qiita jobsの企業ページと相互リンクが張れるなども連携としてありかなと考えています。

**Reply by tanaka-asami415** [reply]
@two2mi 
こちらこそありがとうございます！🙏
今回いただいたコメントも、貴重なご意見として参考にさせていただきます！🙇✨

**Reply by tanaka-asami415** [reply]
@two2mi 
> Qiita jobsの企業ページと相互リンクが張れるなども連携

追記です🙏こちら、jobsの企業管理画面からの設定となりますが可能です！

詳細はこちらをご確認ください🙇
[Qiita Organizationを連携する](https://qiita-jobs-employers.zendesk.com/hc/ja/articles/4865731898767-Qiita-Organization%E3%82%92%E9%80%A3%E6%90%BA%E3%81%99%E3%82%8B)

**Reply by two2mi** [reply]
@tanaka-asami415 
おお、連携できるのですね！参考にします。
